server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailGetMethodContract.scala

```diff
@@ -5995,4 +5994,134 @@ trait EmailGetMethodContract {
            |    "header:To:asAddresses": []
            }""".stripMargin)
   }
+
```

Comment:
Can you write a test of 'group flattening' with :asAddress ?

Reply:
i think it's already been covered by the "asAddressesShouldIgnoresGroup" test case in previous PR?